config/pitest-suppressions/pitest-coding-2-suppressions.xml

```diff
@@ -27,6 +27,30 @@
     <lineContent>throw new IllegalStateException(ILLEGAL_TYPE_OF_TOKEN + ast);</lineContent>
   </mutation>
 
+  <mutation unstable="false">
+    <sourceFile>IllegalInstantiationCheck.java</sourceFile>
```

Comment:
please cover by tests, similar on how same is covered for old values.
We do not allow pitest mutations.